app/helpers/rules_of_origin_helper.rb

```diff
@@ -18,10 +18,6 @@ def rules_of_origin_schemes_intro(country_name, schemes)
       render 'rules_of_origin/intros/country',
              country_name:,
              scheme: schemes.first
-    elsif schemes.first.unilateral
```

Comment:
Note: after the changes, the `unilateral_trade_bloc` partial turned equal to the `trade_bloc`,
therefore I removed it (and reused the trade_bloc).